community/pdns/APKBUILD

```diff
@@ -36,12 +36,14 @@ subpackages="$pkgname-doc $pkgname-openrc $pkgname-tools
 pkgusers="pdns"
 pkggroups="pdns"
 source="https://downloads.powerdns.com/releases/$pkgname-$pkgver.tar.bz2
-	libressl-2.7.patch
 	$pkgname.initd
 	$pkgname.conf"
 builddir="$srcdir/$pkgname-$pkgver"
 
 # secfixes:
+#   4.1.5-r0:
+#     - CVE-2018-10851
+#     - CVE-2018-14626
```

Comment:
would be great to add this top commit message

Reply:
It's not clear to me what I'm requested to do. The top comment has already been edited, is there anything preventing this from getting merged?

Reply:
@andypost are the CVE numbers required in the commit message to get this merged?
I would have to change all pending 10 PRs...

Reply:
IIRC recommented

Reply:
Okay, I've added them.